### PR TITLE
Refactor/merchant

### PR DESCRIPTION
**Pull request by yusualhashash**

### CODE REVIEW

**Review by a0m0rajab — APPROVED** [review]
LGTM 👍 
Thanks.

File: apps/docs/auth-action.ts
Comment (a0m0rajab) [inline apps/docs/auth-action.ts:1]: Did we use the docs/auth-action.ts file?

**Review by a0m0rajab — CHANGES_REQUESTED** [review]

File: apps/docs/auth-action.ts
Comment (yusualhashash) [reply apps/docs/auth-action.ts:1]: yes  i only added th merchantservice to scope for the token

Comment (a0m0rajab) [reply apps/docs/auth-action.ts:1]: This file is in the docs app, we did not use that app for the development.

**Pull request closed without merging**